rustls/tests/api.rs

```diff
@@ -5705,3 +5705,27 @@ fn test_client_removes_tls12_session_if_server_sends_undecryptable_first_message
         ClientStorageOp::RemoveTls12Session(_)
     ));
 }
+
+#[cfg(feature = "ring")]
+#[test]
+fn test_client_fips_service_indicator() {
+    assert_eq!(make_client_config(KeyType::Rsa).fips_mode(), false);
```

Comment:
Style nit: I think we usually want to use `assert!()` directly for boolean values?